botorch/models/cost.py

```diff
@@ -6,6 +6,12 @@
 
 r"""
 Cost models to be used with multi-fidelity optimization.
+
+Cost models are deterministic models, useful for defining known cost functions
```

Comment:
I don't think cost models necessarily need to be deterministic, they could also be probabilistic. In fact, there are some applications in which we make cost observations and model them with a GP model. It just so happens that the models in this module are deterministic.

Reply:
Thanks! `class AffineFidelityCostModel(DeterministicModel)` is the only class in the module so there's no need to be overly general in the module docstring. I'll move some of this material out of here and integrate into the `AffineFidelityCostModel` docstring.